packages/spruce-skill/package.json

```diff
@@ -1,89 +1,89 @@
-	"publishConfig": {
-		"access": "public"
-	},
-	"description": "The base template that drives your Sprucebot Skill! ⚙️🌲🤖",
-	"version": "8.0.2",
-	"homepage": "https://github.com/sprucelabsai/workspace.sprucebot-skills-kit",
-	"bugs": {
-		"url": "https://github.com/sprucelabsai/workspace.sprucebot-skills-kit/issues"
-	},
-	"keywords": [
-		"skills-kit",
-		"sprucebot",
-		"sprucelabs"
-	],
-	"engines": {
-		"node": "^8.5.0",
-		"npm": "^5.4.2"
-	},
-	"scripts": {
-		"build": "next build interface",
-		"start": "node ./server/server.js",
-		"local": "nodemon server/server.js --exec \"node --inspect=4006\" --watch server --watch config --verbose --ignore server/swagger --ignore server/static",
-		"interface": "next ./interface -p 3006",
-		"lint": "./node_modules/.bin/eslint '**/**/*{.js}'",
-		"test": "yarn run build && jest",
-		"test:server": "PORT=3010 BLUEBIRD_W_FORGOTTEN_RETURN=0 ALLOW_NON_SSL=true ORM_LOGGING=false TESTING=true mocha --exit 'server/tests/**/*Tests.js'",
-		"test:server:single": "PORT=3010 BLUEBIRD_W_FORGOTTEN_RETURN=0 ALLOW_NON_SSL=true ORM_LOGGING=false TESTING=true mocha --exit",
-		"icon": "./node_modules/.bin/svgo icon/icon.svg",
-		"precommit": "npm run lint",
-		"prepush": "npm run lint && npm run test",
-		"heroku-postbuild": "npm run build",
-		"#postinstall": "npm run build",
-		"release": "semantic-release"
-	},
-	"dependencies": {
-		"@babel/runtime": "^7.0.0",
-		"@sprucelabs/log": "^1.6.2",
-		"@sprucelabs/react-heartwood-components": "^8.0.2",
-		"@sprucelabs/spruce-next-helpers": "^8.0.2",
-		"@sprucelabs/spruce-node": "^8.0.2",
-		"@sprucelabs/spruce-skill-server": "^8.0.2",
-		"@zeit/next-css": "^1.0.1",
-		"@zeit/next-sass": "^1.0.1",
-		"babel-core": "^7.0.0-bridge.0",
-		"config": "^1.27.0",
-		"dotenv": "^5.0.1",
-		"enzyme": "^3.3.0",
-		"enzyme-adapter-react-16": "^1.1.1",
-		"enzyme-to-json": "^3.3.1",
-		"eslint-plugin-react": "^7.4.0",
-		"globby": "^8.0.1",
-		"ioredis": "^4.0.1",
-		"jsonwebtoken": "^8.1.0",
-		"lodash": "^4.17.4",
-		"moment-timezone": "^0.5.14",
-		"next": "^7.0.0",
-		"node-sass": "^4.9.3",
-		"prop-types": "^15.6.0",
-		"react": "^16.0.0",
-		"react-dom": "^16.0.0",
-		"react-emojione": "^5.0.0",
-		"react-redux": "^5.0.6",
-		"redux": "^3.7.2",
-		"swagger-jsdoc": "^3.0.2",
-		"universal-cookie": "^2.1.0"
-	},
-	"devDependencies": {
-		"@semantic-release/changelog": "^3.0.0",
-		"@semantic-release/git": "^7.0.1",
-		"@sprucelabs/semantic-release": "^1.0.1",
-		"babel-jest": "^23.6.0",
-		"conventional-changelog-sprucelabs": "^1.1.0",
-		"eslint": "^4.9.0",
-		"eslint-config-prettier": "^2.9.0",
-		"eslint-plugin-prettier": "^2.3.1",
-		"husky": "^0.14.3",
-		"jest": "^22.0.5",
-		"jest-cli": "^22.0.5",
-		"nock": "^9.2.3",
-		"nodemon": "^1.18.7",
-		"prettier": "^1.13",
-		"raf": "^3.4.0",
-		"semantic-release": "^15.10.6",
-		"svgo": "^1.0.3"
-	},
-	"gitHead": "eac8d4ea6df58218b78c9c866b22c761df1e6447"
+  "name": "@sprucelabs/spruce-skill",
```

Comment:
Did you mean to un-tab this? (Thought yarn respected .eslintrc when adding things)